src/workloads/scale/LLTQuery.yml

```diff
@@ -0,0 +1,421 @@
+  # the workload runs consistently.
+  # Not In-memory: Database size works out about 30GB.
+  # InitialDocumentCount: &InitialNumDocs 49000000
+  # SecondaryDocumentCount: &SecondaryNumDocs 1000000
+  CollectionCount: &CollectionCount 4
+
+  PdcQueryOperation: &PdcQueryOperation
+    OperationName: findOne
+    OperationCommand:
+      Filter: &PdcFilter {price: {'$gte' : {^RandomDouble: { min: 0.0, max: 500.0 }}}}
+      Options: &PdcQueryOperationOptions
+        Hint: price_1_ts_1_cuid_1
+        Comment: PdcQueryOperation
+
+  MarketQueryOperation: &MarketQueryOperation
+    OperationName: findOne
+    OperationCommand:
+      Filter: &MarketFilter {'price': {'$gte' : {^RandomDouble: { min: 0.0, max: 500.0 }}}}
+      Options: &MarketQueryOperationOptions
+        Hint: price_1_cuid_1
+        Comment: MarketQueryOperation
+
+  RegisterQueryOperation: &RegisterQueryOperation
+    OperationName: findOne
+    OperationCommand:
+      Filter: &RegisterFilter {'caid':  {'$gte': {^RandomInt: { min: 0, max: 1000 }}}}
+      Options: &RegisterQueryOperationOptions
+        Hint: caid_1_price_1_cuid_1
+        Comment: RegisterQueryOperation
+
+  # Low Baseline / Benchmark
+  LowGlobalRate: &LowGlobalRate 400 per 1 second
+  LowThreads: &LowThreads 8
+
+  # Low Baseline / Benchmark
+  HighGlobalRate: &HighGlobalRate 800 per 1 second
+  HighThreads: &HighThreads 16
+
+  # Scanner
+  SnapshotScannerShortDuration: &SnapshotScannerShortDuration 2 minutes
+  SnapshotScannerMediumDuration: &SnapshotScannerMediumDuration 10 minutes
+  SnapshotScannerLongDuration: &SnapshotScannerLongDuration 30 minutes
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: -1
+      maxPoolSize: 200
+  HighPool:
+    QueryOptions:
+      socketTimeoutMS: -1
+      maxPoolSize: 400
+
+# Odd phases do operations, even phases quiesce.
+Actors:
+- Name: InitialLoad
+  Type: Loader
+  ClientName: Loader
+  Threads: *LoadThreads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Threads: *LoadThreads
+        CollectionCount: *CollectionCount
+        Database: *dbname
+        Repeat: 1
+        Document: *Doc
+        DocumentCount: *InitialNumDocs
+        Indexes: *LLTIndexes
+        BatchSize: *LoadBatchSize
+
+- Name: SecondLoadAfterIndexes
+  Type: Loader
+  ClientName: Loader
+  Threads: *LoadThreads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Threads: *LoadThreads
+        CollectionCount: *CollectionCount
+        Database: *dbname
+        Repeat: 1
+        Document: *Doc
+        DocumentCount: *SecondaryNumDocs
+        BatchSize: *LoadBatchSize
+
+- Name: QuiescePhase
+  Type: QuiesceActor
+  Threads: 1
+  Database: *dbname
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20]
+      NopInPhasesUpTo: 20
+      PhaseConfig:
+        Repeat: 1
+
+- Name: UpdateBaseline.Low.50
```

Comment:
I think you want to update the actor name here.

Reply:
:+1:  more cut and paste, I thought I had fixed this already .